### PR TITLE
Add two challenges to 10-open.md

### PR DESCRIPTION
**Pull request by datamat**
[This is part of the instructor training checkout procedure]

I added two new challenges to make the students more familiar with (1) the various open access data repositories out there and (2) how it's possible to also make your code citable, not just data. I believe this supports the open science objective (Explain how a version control system can be leveraged as an electronic lab notebook for computational work) by highlighting the many possibilities the share information. 

Looking forward to the Bluejeans session.

### CODE REVIEW

File: 10-open.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Version Control with Git
 subtitle: Open Science
-minutes: 5
+minutes: 5-10
```
Comment (iglpdc) [inline 10-open.md]: You can put 10 minutes for this: 5 to explain the lesson and 5 to do a challenge.

File: 10-open.md [hunk 2]
```diff
@@ -104,3 +104,22 @@ by acting as a shareable electronic lab notebook for computational work:
...
+> You might also find this page very useful -> [Nature: Recommended Data
+> Repositories](http://www.nature.com/sdata/data-policies/repositories). 
+> Discuss with your neighbor which data repository you might want to
+> approach for your current project and explain why. 
+
+> ## Can I also publish code? {.challenge}
+>
+> There are many new ways to publish code and to make it citable. One
+> way is described [here](https://guides.github.com/activities/citable-code/).
+> Basically it's a combination of github (where the code is) and zenodo (the
```
Comment (iglpdc) [inline 10-open.md]: 'github' -> 'GitHub' and 'zenodo' -> 'Zenodo'

File: 10-open.md [hunk 3]
```diff
@@ -104,3 +104,22 @@ by acting as a shareable electronic lab notebook for computational work:
...
 > Try to do the same for one of their results,
 > then try to do it for a result from a lab you work with.
+
+> ## How to find an appropriate data repository? {.challenge}
+>
+> Surf the internet for a couple of minutes and check out the data repositories
+> mentioned above: [Figshare](http://figshare.com/), [Zenodo](http://zenodo.org), 
+> [Dryad](http://datadryad.org/). Depending on your field of research, you might 
+> find community-recognized repositories that are well-known in your field.
+> You might also find this page very useful -> [Nature: Recommended Data
```
Comment (iglpdc) [inline 10-open.md]: Could you incorporate the link into the sentence? Like "You might find useful [these data repositories recommended by Nature](link)"

**Comment by iglpdc** [comment]
Thanks @datamat ! Could you take a look to my suggestions above?

File: 10-open.md (hunk 3 above)
Comment (datamat) [inline 10-open.md]: Dear Ivan,

 

Thanks for your suggestions, I changed the challenges accordingly. Should I make a new pull request?

 

Best wishes,
Matthias

**Comment by gvwilson** [comment]
@datamat can you please revise as requested and then we'll merge?

**Comment by datamat** [comment]
Done! Thanks and happy merging.

**Comment by iglpdc** [comment]
Hey @datamat, thanks again for your contribution. This PR cannot be merged as it was because a conflict with some other stuff I merged previously, so I grabbed your commits and make a new PR and merged after rebasing it. 

See #297 to see your commits in. I'm closing this.

**Pull request closed without merging**